subsys/logging/Kconfig

```diff
@@ -285,5 +285,33 @@ config LOG_BACKEND_UART_FORMAT_TIMESTAMP
 	default y
 	help
 	  When enabled timestamp is formatted to hh:mm:ss:ms,us.
+
+config LOG_BACKEND_RTT
+	bool "Enable RTT backend"
+	depends on RTT_CONSOLE
+	default y
+	help
+	  When enabled backend is using RTT to output logs.
+
+config LOG_BACKEND_RTT_BUFFER_SIZE
```

Comment:
you can wrap all 3 options into 
`if LOG_BACKEND_RTT`
and remove 'depends on ..' (btw buffer size is missing 'depends on')